Pipfile

```diff
@@ -4,20 +4,20 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+
+[dev-packages]
 docutils = "*"
-MarkupSafe = "*"
-Pygments = "*"
-pylint = "*"
 pytest = "*"
 pytest-cov = "*"
 pytest-pylint = "*"
+MarkupSafe = "*"
+Pygments = "*"
+pylint = "*"
 python-dotenv = ">=0.5.1"
 Sphinx = "*"
 sphinx_rtd_theme = "*"
 sphinx-autobuild = "*"
 sphinxcontrib-websupport = "*"
 
-[dev-packages]
-
 [requires]
-python_version = "3"
```

Comment:
`python_version = "3"` should remain, otherwise `pipfile` will fail when installing packages in the local machine